xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -3348,10 +3628,24 @@ The property name and value is escaped before writing.
         <Parameter Name="utf8Value" Type="System.ReadOnlySpan&lt;System.Byte&gt;" />
       </Parameters>
       <Docs>
-        <param name="propertyName">To be added.</param>
-        <param name="utf8Value">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="propertyName">The JSON encoded property name of the JSON object to be transcoded and written as UTF-8.</param>
+        <param name="utf8Value">The UTF-8 encoded value to be written as a JSON string as part of the name/value pair.</param>
+        <summary>Writes the pre-encoded property name and UTF-8 text value (as a JSON string) as part of a name/value pair of a JSON object.</summary>
+        <remarks>
+          <format type="text/markdown">
+            <![CDATA[
+
+## Remarks
+
+The property name should already be escaped when the instance of <xref:System.Text.Json.JsonEncodedText> was created.
+
+The value is escaped before writing.
```

Comment:
This remark section looks much better (we should follow the same structure in other similar cases that I mentioned above).

Just to clarify, this is an issue/mistake with the comments coming from source (and nothing to do with tools or this port itself). They all weren't properly formatted with `<para>` for multiple remarks to show up nicely (though some were fix, like this one).

https://github.com/dotnet/corefx/blob/306fe9d60cc6db02a69d8ea6b88dde2b98fe8a7c/src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.SignedNumber.cs#L51-L54

However, note that the first remark was removed in https://github.com/dotnet/corefx/pull/39092/files#diff-71ab5a4f550eede8ad13896c013094cfL501 and we should remove it here as well (this feedback could be resolved by refreshing the port, for `Utf8JsonWriter.xml` specifically).

Reply:
@ahsonkhan The remark is still there in your second hyperlink. It has not been removed from source. Can you clarify this?

Reply:
Let's address this in a separate PR.

Reply:
It was removed: https://github.com/dotnet/corefx/blob/64a9e2ccf5e590e0a3a3d6dc9c2a7c828850c912/src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.String.cs#L499-L503

![image](https://user-images.githubusercontent.com/6527137/61901725-44f9cf00-aed5-11e9-9eb8-a383a438cd41.png)

Addressing it separately works for me. Tyvm for your help with this.